packages/terra-consumer-nav/src/NavPropShapes.js

```diff
@@ -22,6 +22,10 @@ export const navItemShape = {
    */
   isActive: PropTypes.bool,
   /**
+   * Whether or not the link is external.
+   */
+  isExternal: PropTypes.bool,
+  /**
    * An optional badge. When supplied, displays the value inline, styled alongside the text.
    */
   badgeValue: PropTypes.number,
```

Comment:
@KevinJJackson added a default props section. Should isExternal be added there?

Reply:
it seems the default props section defined in this file is never used. Since isExternal is only used in smartlink, i prefer setting the default there

Reply:
it is currently not used and was created in advance of the targets. If it's determined they aren't needed and each component should default their own then feel free to remove it, otherwise they should be applied to navItems, profileLinks and navHelp.

I agree that anything specific to a component like isExternal or badgeValue for nav-items should not be in the default structure and should be left to the component.